PENDING_RELEASE_NOTES.md

```diff
@@ -1,14 +0,0 @@
-
-## Version 16.2.0
-_Date TBD_
-
-### New Features
```

Comment:
These need to come back. The are the PENDING RELEASE NOTES, as they're on STG right now